Havit.Blazor.Components.Web.Bootstrap.Documentation/Pages/Components/Concepts/SettingsAndDefaults_Demo_Settings.razor

```diff
@@ -0,0 +1,11 @@
+﻿<HxButton Settings="settings" Text="Fullscreen" />
+<HxButton Settings="settings" Text="Open" />
+
+@code {
+    private ButtonSettings settings = new()
```

Comment:
Settings demo should show use-case where using the settings adds some value.
* define several settings in standalone file, such as
```csharp
public static class ButtonSettings
{
  public static ButtonSettings MainButton { get; } = HxButton.Defaults with
  {
     Color = ThemeColor.Primary
  };
  public static ButtonSettings SecondaryButton { get; } = HxButton.Defaults with
  {
     Color = ThemeColor.Secondary,
      Outline = true,
  };
}
```

...and then demonstrate usage of such settings pre-sets in page.